### ISSUE TITLE
Conflicts with vscode-styled-component

### ISSUE THREAD

**Issue by u3u** [Divlo/vscode-styled-jsx#6]
## Steps To Reproduce

1. Installation this plugin
2. Installation [vscode-styled-component](https://marketplace.visualstudio.com/items?itemName=jpoissonnier.vscode-styled-components)

## The current behavior

```jsx
import { css } from '@emotion/core';

// error: } expectedcss(css-rcurlyexpected)
// and syntax highlighting is broken
// it should conflict with styled-jsx's `css.resolve` and `css.global`
const className = css`
  text-align: center;
`;

const Button = () => {
  return (
    <button
      // same problem as above
      css={css`
        text-align: center;
      `}
    />
  );
};
```

Hope this conflict can be resolved 🙏

**Comment by Divlo** [comment]
@u3u Yes, that's a known issue, we can't do anything about that (as I'm aware).
But as a workaround, you can disable/enable the 2 extensions. If you're working with styled-jsx components, enable this extension and disable the other one and the contrary if you're working with styled-components.

**Comment by u3u** [comment]
Because of [this](https://github.com/vercel/styled-jsx/issues/573) problem, i usually combine the two, so it is a trouble for me 😢

**Comment by kud** [comment]
I've got the same problem. I've disabled this plugin for the moment.
